package.json

```diff
@@ -0,0 +1,75 @@
+{
+  "name": "device-id-widget",
+  "widgetName": "DeviceIdWidget",
+  "version": "1.0.0",
```

Comment:
yes, during dev we keep it 0.1.0 only at end of sprint we update the number.. but it ok to keep it. 1.0.0